db/schema.rb

```diff
@@ -226,25 +226,34 @@
   add_index "devices", ["deleted_at"], name: "index_devices_on_deleted_at", using: :btree
 
   create_table "encounters", force: :cascade do |t|
-    t.integer  "institution_id",  limit: 4
-    t.integer  "patient_id",      limit: 4
-    t.string   "uuid",            limit: 255
-    t.string   "entity_id",       limit: 255
-    t.binary   "sensitive_data",  limit: 65535
-    t.text     "custom_fields",   limit: 65535
-    t.text     "core_fields",     limit: 65535
+    t.integer  "institution_id",    limit: 4
+    t.integer  "patient_id",        limit: 4
+    t.string   "uuid",              limit: 255
+    t.string   "entity_id",         limit: 255
+    t.binary   "sensitive_data",    limit: 65535
+    t.text     "custom_fields",     limit: 65535
+    t.text     "core_fields",       limit: 65535
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.boolean  "is_phantom",                    default: true
+    t.boolean  "is_phantom",                      default: true
     t.datetime "deleted_at"
-    t.integer  "site_id",         limit: 4
+    t.integer  "site_id",           limit: 4
     t.datetime "user_updated_at"
-    t.string   "site_prefix",     limit: 255
+    t.string   "site_prefix",       limit: 255
     t.datetime "start_time"
+    t.integer  "user_id",           limit: 4
```

Comment:
I think they are used in app/assets/javascripts/components/fresh_tests_encounter_form.js.jsx